### DISCUSSION TITLE
Dev em BH e região

### DISCUSSION THREAD

**Discussion by welbertdeoliveira** [RamonSilva20/mapos#2060] [Q&A]
Boa noite. Estamos utilizando o Map-OS na nossa empresa e tem sido muito útil. 

Entretanto, necessitamos fazer algumas melhorias na aplicação e gostaríamos de contratar algum dev aqui região pra fazer essas implementações. 

Caso haja alguém que se interesse pelo trabalho, agradeceria o retorno. 

Obrigado.

**Comment by Marcosrocha5858** [comment]
Olá Welbert , sou programador php , já atuei neste projeto
posso fazer melhorias de acordo com sua necessidade, meu e-mail: marcosrocha5858@gmail.com

**Reply by welbertdeoliveira** [reply]
Perdoe a demora. Retornei seu email. Quando possível, por favor, peço entrar em contato. Obrigado.

**Comment by Marcosrocha5858** [comment]
Claro , me chama no WhatsApp 27997352126 , posso atender no que vocês
precisarem

Em ter., 9 de mai. de 2023 19:04, Welbert de Oliveira <
***@***.***> escreveu:
